src/color-legend/ColorLegend.js

```diff
@@ -67,8 +67,8 @@ export default function ColorLegend({ highlighted, items, setHighlighted }) {
           key={label}
           onBlur={() => setHighlighted()}
           onFocus={() => setHighlighted({ label })}
-          onMouseOut={() => setHighlighted()}
-          onMouseOver={() => setHighlighted({ label })}
+          onMouseLeave={() => setHighlighted()}
+          onMouseEnter={() => setHighlighted({ label })}
```

Comment:
FYI: there were some performance issues with the new chart and this change helped; these new events are triggered at the same time and have the same surface but do not bubble up through the DOM